src/mongocxx/collection.hpp

```diff
@@ -329,10 +318,8 @@ class MONGOCXX_API collection {
     ///
     /// @see http://docs.mongodb.org/manual/core/read-operations-introduction/
     ///
-    bsoncxx::stdx::optional<bsoncxx::document::value> find_one(
-        bsoncxx::document::view filter,
-        const options::find& options = options::find()
-    );
+    stdx::optional<bsoncxx::document::value> find_one(
+        bsoncxx::document::view filter, const options::find& options = options::find());
```

Comment:
Yes! After removing the `bsoncxx::` clang-format wanted them on the same line. I guess it shrunk just enough? Either that or it was breaking the format before.